packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -236,7 +244,6 @@ function s(
                 }
                 // If the passed slot content is factory, evaluate it and add the produced vnodes
                 if (assignedNodeIsScopedSlot) {
-                    const vmBeingRenderedInception = getVMBeingRendered();
```

Comment:
What is the reason for this change?

Reply:
This was to hoist the `vmBeingRendered` to the :top: since we're using it in the rest of the function.